### DISCUSSION TITLE
이건 정말 잘못된 설계였다

### DISCUSSION THREAD

**Discussion by leehosung** [8percent/rdb-study#22] [Ideas]
자신의 잘못된 설계로 고생 했던 경험을 나눠 보면 어떨까?

**Comment by wer153** [comment]
다대다 관계(many to many field)의 모델들을 정의한 적이 있었어요.
연결 테이블을 지정해주지 않아서 ORM으로 원하는 데이터를 추출하는 작업이 까다로웠어요.
일대다관계의 테이블을 연결 테이블로 지정하니, 개념적인 이해도 쉽고 ORM활용이 쉬워졌습니다.
